test/ethereum-identity-provider.test.js

```diff
@@ -0,0 +1,146 @@
+'use strict'
+
+const assert = require('assert')
+const path = require('path')
+const rmrf = require('rimraf')
+const mkdirp = require('mkdirp')
+const LocalStorage = require('node-localstorage').LocalStorage
+const Keystore = require('orbit-db-keystore')
+const IdentityProviders = require('../src/identity-providers')
+const EthIdentityProvider = require('../src/ethereum-identity-provider')
+const Identity = require('../src/identity')
+const savedKeysPath = path.resolve('./test/fixtures/keys')
+const keypath = path.resolve('./test/keys')
+let keystore
+
+const type = EthIdentityProvider.type
+describe('Ethereum Identity Provider', function() {
+
+  before(async () => {
+    rmrf.sync(keypath)
+    IdentityProviders.addIdentityProvider({ IdentityProvider: EthIdentityProvider })
```

Comment:
Do we need the `IdentityProvider` field here or could we just pass the class `EthIdentityProvider`? Are there other expected options to be passed to `addIdentityProvider()`